### PR TITLE
Grants

### LINKED ISSUE #97: Add `grants` to dbt_project.yml model configs
When adding grants to `dbt_project.yml` you curently see "Incorrect type.  Expected "null".

image

### PR DESCRIPTION
**Pull request by dave-connors-3**
closes #97

only for you @dpguthrie

### CODE REVIEW

**Comment by dpguthrie** [comment]
What a guy!

**Comment by joellabes** [comment]
@dave-connors-3 now that #94 is merged, there's some conflicts sorry!

**Comment by dave-connors-3** [comment]
@joellabes hit me with that sweet sweet review

File: schemas/1.5/dbt_project-1.5.json [hunk 1]
```diff
@@ -818,6 +830,16 @@
         "additionalProperties": false
       }
     },
+    "grants": {
+      "type": "object",
+      "description": "flexible grant config. accepts any string key, expects a string or list of strings value",
```
Comment (joellabes) [inline schemas/1.5/dbt_project-1.5.json:835]: The various IDEs should take responsibility for saying what the type is. The description would be better if it was what the strings actually are (what is granted, to whom it is granted)

**Review by joellabes — APPROVED** [review]
A tweak everywhere it comes up, but once that's sorted ship it!

**Review by joellabes — DISMISSED** [review]
❤️

**Review by joellabes — APPROVED** [review]
The diff for schemas/dbt_yml_files.json looks enormous but if you disable whitespace diffing it is a normal and good amount of changes. And the whitespace in this change is actually the correct amount.

**Pull request merged**